tests/utils/config/config.go

```diff
@@ -183,6 +183,10 @@ func readFile(cfg *Config, cfgFile string) error {
 		return err
 	}
 
+	cfg.General.TnfConfigDir = os.ExpandEnv(cfg.General.TnfConfigDir)
+	cfg.General.TnfReportDir = os.ExpandEnv(cfg.General.TnfReportDir)
+	cfg.General.DockerConfigDir = os.ExpandEnv(cfg.General.DockerConfigDir)
```

Comment:
These aren't needed because we are already using `envconfig` for these vars.

Reply:
Doesn't work without it. What is envconfig?